### PR TITLE
Fixing change order available to read only admin

### LINKED ISSUE #457: [BUG] Change order of Service offering is available to Read-only admin
**Describe the bug**
With the latest default role introduced - read only admin he's able to see the buttons for changing the order of * offerings
When click on some of the buttons he's immediately logged out. 

As per the autodisovery nature of this UI I believe users shouldn't be able to see such buttons, not sure if there's a solution on a framework level but it'll be good to look into such instead of looking into each custom role and logging each button that shouldn't be available to user. 

readonly

### PR DESCRIPTION
**Pull request by davidjumani**
Fixes #457

### CODE REVIEW

**Comment by shwstppr** [comment]
@blueorganutan package

**Comment by shwstppr** [comment]
@blueorangutan package

**Comment by davidjumani** [comment]
@blueorangutan package

File: src/components/view/ListView.vue [hunk 1]
```diff
@@ -275,6 +275,12 @@ export default {
     }
   },
   methods: {
+    fetchColumns () {
```
Comment (rhtyd) [inline src/components/view/ListView.vue:278]: Usually we would want a computed property, but this works

**Pull request merged**